### PR TITLE
Fatnaney1 add caption

### PR DESCRIPTION
**Pull request by fatnaney1**
Please review the changes

### CODE REVIEW

**Comment by kostyanyc** [comment]
Looks nice.....

File: _posts/0000-01-12-fatnaney1.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome to our slide deck- Naina I love !"
```
Comment (kostyanyc) [inline _posts/0000-01-12-fatnaney1.md]: Cool Comment

**Review by v421vakil — APPROVED** [review]
looks good

**Review by kostyanyc — APPROVED** [review]
Approve this request..

**Pull request merged**